README.md

```diff
@@ -1,40 +1,67 @@
 # ZenDesk App
 
-## Quick Start
+## Development Repository Branches Needed
 
-- clone this repo
-- cp `.env.sample` `.env`
-  - add your credentials to the `.env` file
-    - ZENDESK_URL - URL to zendesk account
-    - ZENDESK_USERNAME - your zendesk login username
-    - ZENDESK_API_TOKEN - generated via the following steps
-      - zendesk > Admin > CHANNELS > API > Settings
-      - Token Access > Enable
-      - Add API Token
+The current master branches have not been merged to work with this app and the following are needed until they are merged with master
 
-- `npm start` - start the node server
-- `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
-- `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+mm-webapp: `feature/cloud-apps`  
+mm-plugin-apps: `master`
 
-**Current Branches Needed**
+## Quick Start
 
-The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+### Zendesk and Mattermost Users (System Admin privileges required)
+
+1. Clone this repo
+1. cp `.env.sample` `.env`
+    1. `ZENDESK_URL` - set URL to your zendesk account host
+        1. Ex. `https://<subdomain>.zendesk.com`
+    1. `ZENDESK_CLIENT_SECRET` - (will be set later in the setup)
+1. Create Zendesk Oauth Client for Mattermost (in Zendesk)
+    1. `Zendesk` > `Admin` > `API` > `OAuth Clients`
+    1. `Add OAuth Client`
+        1. `Client Name`: `Mattermost Zendesk App`
+        1. `Description`: `Connect your Zendesk to accout to Mattermost`
+        1. `https://<your-zendesk-app-host>/mattermost/oauth/complete`
+            1. Ex. `https://localhost:4000` - Development
+            1. Ex. `https://mytest.ngrok.io` - Exposed for development
+        1. `Secret` - Save the generated secret in `.env` as the `ZENDESK_CLIENT_SECRET`
+        1. `Save`
+1. Start the node server
+    1. `npm start` - start the node server
+    1. `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
+    1. `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+1. Install the app (In Mattermost)
+    1. `/apps install --url http://<your-zendesk-app-host>/mattermost/manifest.json --app-secret thisisthesecret`  
```

Comment:
If thisisthesecret is zendesk-client-secret, might be helpful to call it as such. Also, we should consolidate on one of : your-zendesk-app-host, your-zendesk-app and zendesk-url.

Reply:
I'm not currently doing anything with the --app-secret at this time.  I need to understand the install better, but I think this slash command is temporary. Will create a ticket

Reply:
ticket created https://mattermost.atlassian.net/browse/MM-31907